src/aliceVision/calibration/checkerDetector.cpp

```diff
@@ -1148,19 +1151,40 @@ void CheckerDetector::drawCheckerBoard(image::Image<image::RGBColor> & img) cons
             for (int j = 0; j < board.cols(); j++)
             {
                 const IndexT p1 = board(i, j);
-                const IndexT p2 = board(i + 1, j);
+                IndexT p2 = board(i + 1, j);
```

Comment:
this loop is suspiciously similar to the previous one.
Maybe the two can be refactored and/or better put in a helper function. 
This would also improve the readability